resources/wikia/modules/domCalculator.js

```diff
@@ -0,0 +1,25 @@
+/*global define*/
+define('wikia.domCalculator', [
+], function () {
+	'use strict';
+
+	function getTopOffset(element) {
+		var topPos = 0,
```

Comment:
Then we need to move assigning `element.offsetParent` as well, so basically this is what `do...while` does without duplicating loop body